.github/workflows/docker_build.yml

```diff
@@ -7,25 +7,37 @@ on:
   push:
     branches: ['release*']
 
+env:
+  REGISTRY: ghcr.io
+  IMAGE_PATH: ghcr.io/${{ github.repository }}
+
 jobs:
-  build-and-push-to-dockerhub:
+  build-and-push-image:
     runs-on: ubuntu-latest
-    steps:
-      - name: Check out the repo
-        uses: actions/checkout@v3
+    permissions:
+      contents: read
+      packages: write
 
-      - name: Log in to Docker Hub
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+          
+      - name: Log in to the Container registry
```

Comment:
Migrating the base image location would also require updating the instructions to use it: https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2720759990/Deploying+a+Web+Application+to+AWS

Reply:
Thanks for pointing that out. I can update that if we decided to change repositories. I think it's convenient to have the image hosted in the same github repo. I think you've said the same thing when developing the AWS deployment infrastructure.